examples/service.yaml

```diff
@@ -0,0 +1,22 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: kogito-greetings-external
+  labels:
+    app: kogito-greetings
+spec:
+  ports:
+    - protocol: TCP
+      port: 8080
+      targetPort: 8080
+      nodePort: 31961
+  selector:
+    app: kogito-greetings
+  type: LoadBalancer
```

Comment:
isn't better to use NodePort as type?

Reply:
This one was created by minikube 🤔 

I'll change to  NodePort, no reason why not.